### PR TITLE
Remove external nodejs dependency of Git

### PR DESCRIPTION
**Pull request by prietyc123**
**What type of PR is this?**

/kind failing-test

**What does does this PR do / why we need it**:
PR fix the failure https://github.com/openshift/odo/pull/4063#issuecomment-768911222

**Which issue(s) this PR fixes**:

Fixes

**PR acceptance criteria**:

- [ ] Unit test 

- [x] Integration test 

- [ ] Documentation 

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:
All the tests should pass.

### CODE REVIEW

**Review by mohammedzee1000 — APPROVED** [review]
Codewise looks good
/lgtm

File: tests/integration/cmd_push_test.go [hunk 1]
```diff
@@ -330,8 +336,8 @@ var _ = Describe("odo push command tests", func() {
 				},
 			)
 
-			Expect(modifiedCatViewFile).NotTo(Equal(earlierCatViewFile))
-			Expect(modifiedCatServerFile).To(Equal(earlierCatServerFile))
+			Expect(modifiedCatPackageFile).To(Equal(earlierCatPackageFile))
```
Comment (girishramnani) [inline tests/integration/cmd_push_test.go:339]: why is it other other way around - before it was `To` and now its `NotTo`?

Comment (prietyc123) [reply tests/integration/cmd_push_test.go:339]: Because this time I am doing changes in server file, not in package.json

**Comment by dharmit** [comment]
/hold

Till #4399 is merged.

**Review by dharmit — APPROVED** [review]
/lgtm

**Review by mohammedzee1000 — COMMENTED** [review]
/lgtm

**Comment by dharmit** [comment]
> /hold
> 
> Till #4399 is merged.

/hold cancel

**Pull request merged**